test/mpi/distributed/vector.cpp

```diff
@@ -420,11 +418,7 @@ class VectorReductions : public ::testing::Test {
         y = gko::clone(exec, tmp_y);
     }
 
-    void SetUp() override
-    {
-        ASSERT_GT(comm.size(), 0);
-        init_executor(gko::ReferenceExecutor::create(), exec);
-    }
+    void SetUp() override { ASSERT_GT(comm.size(), 0); }
```

Comment:
What's the reason for these changes?

Reply:
Because I would need to re-add the logger to exec here, but instead I just do it once in the constructor.